src/cleanfigure.m

```diff
@@ -484,9 +485,12 @@ function simplifyLine(meta, handle, targetResolution)
                 mask      = opheimSimplify(x, y, tol);
                 % Remove all those with mask==0 respecting the number of
                 % data points in the previous segments
-                id_remove = find(mask==0) + lineStart(ii) - 1;
+                id_remove{ii} = find(mask==0) + lineStart(ii) - 1;
```

Comment:
Hm not sure if this is correct. This is still the individual line segment. So we need to translate the mask to indices and then add the line segments up.